config/config_test.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_LoadFromPath(t *testing.T) {
```

Comment:
I think it would also be good to have a test that verifies that even when the file doesn't exist it returns a valid config.

Reply:
I have added this test case as suggested :+1: